references/classification/train.py

```diff
@@ -68,8 +68,11 @@ def load_data(data_path, use_OpenFire=False, download=False, img_size=224, crop_
 def main(args):
 
     print(args)
+    if args.use-openfire:
+        use-openfire=True
 
-    train_set, val_set = load_data(args.data_path, use_OpenFire=args.use_OpenFire, img_size=args.img_size)
+
+    train_set, val_set = load_data(args.data_path, use_openfire=use-openfire, img_size=args.img_size)
```

Comment:
simply pass `args.use_openfire`, no need to complicate this :sweat_smile:

Reply:
WTF?? no idea why I did this ... haha